### PR TITLE
Adding search input group component

### PR DESCRIPTION
**Pull request by NeOMakinG**
Related to https://github.com/PrestaShop/PrestaShop/issues/16598

How to test : Build assets (npm run build or npm run watch) and then compare with issue's specs on the index.html of the UI Kit, if there are var missing, it's because past PR's needs to be merged before..

### CODE REVIEW

File: scss/_search.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    &.input-group {
+        .#{$component-name}-input {
+            padding: 0 15px;
+
+            &:focus {
+                border-color: $primary;
+            }
+        }
+    }
+}
```
Comment (PierreRambaud) [inline scss/_search.scss]:
```suggestion
}

```

**Comment by sarahdib** [comment]
@NeOMakinG 
the rounding on the right side is missing
Capture d’écran 2019-12-19 à 09 21 47

**Comment by NeOMakinG** [comment]
> @NeOMakinG
> the rounding on the right side is missing
> Capture d’écran 2019-12-19 à 09 21 47

Hey Sarah,

the rounding is in this PR... https://github.com/PrestaShop/prestashop-ui-kit/pull/69/files

in _variable.scss

I did'nt do it on most of PR's avec this one to avoid conflicts :/

File: scss/_search.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+$component-name: search;
+
+.#{$component-name} {
+    &.input-group {
+        .#{$component-name}-input {
+            padding: 0 15px;
```
Comment (eternoendless) [inline scss/_search.scss]: Consider using rem instead of px

Comment (PierreRambaud) [reply scss/_search.scss]: You're right, padding must use rem :)

**Pull request merged**